04 - Playing with PyPI.qmd

```diff
@@ -242,25 +298,7 @@ chart = (
 chart
 ```
 
-We can also use `plotnine`!
-
-```{python}
-from plotnine import aes, geom_histogram, ggplot, scale_y_log10, geom_bar
-```
-
-```{python}
-#| scrolled: true
-(
-    ggplot(maintainer_counts, aes(x="maintainers", y="count"))
-    + geom_bar(stat="identity")
-    + scale_y_log10()
-)
-```
-
-Finally, since we can always call `to_pandas` or `to_pyarrow` -- it's easy to
-connect an Ibis result to your favorite PyData tools.
-
-### Exercise 2
+### Exercise 05
```

Comment:
These lines weren't changed in this PR, so commenting here: why do we `repr` the version of `ibis` and `altair` in the codeblock above?

Reply:
Ahh, i can remove that. That was an old version check to make sure Altair could take the ibis expression directly